trio/testing/_fake_net.py

```diff
@@ -33,7 +33,7 @@ def _family_for(ip: IPAddress) -> int:
         return trio.socket.AF_INET
     elif isinstance(ip, ipaddress.IPv6Address):
         return trio.socket.AF_INET6
-    assert False  # pragma: no cover
+    raise AssertionError()  # pragma: no cover
```

Comment:
here

Reply:
Changed in a5afed7 and added reason in e1fdebc8